### DISCUSSION TITLE
What is the "disabled" folder for?

### DISCUSSION THREAD

**Discussion by ghost** [kus/cs2-modded-server#94] [General]
Why are there plugins in a disabled folder when in the readme we could change some files in there to enable e.g. advertising, etc.?
What is you thought behind the "disabled" folder?

**Comment by kus** [comment]
As the server is a multi game mode server you don’t want every plugin loaded when you don’t need them.

The plugins in the plugins folder are all loaded automatically, and these are required for every game mode.

The plugins in disabled are not loaded automatically and you need to manually load them. So when you switch game mode it unloads all the plugins in disabled and loads the plugin for the game mode you are switching to.

Generally the plugins are all setup for each game mode so you don’t need to worry about it. Only if you add something custom or enable a plugin I don’t have on like Advertisements and I explain how to do that [here](https://github.com/kus/cs2-modded-server?tab=readme-ov-file#enable-advertisements).
